src/Core/Core.csproj

```diff
@@ -19,6 +19,10 @@
     <EmbeddedResource Include="MailTemplates\Handlebars\**\*.hbs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="MailTemplates\Handlebars\SecretsManagerAccessRequest.html.hbs" />
+  </ItemGroup>
```

Comment:
I think the embedded resource include `<EmbeddedResource Include="MailTemplates\Handlebars\**\*.hbs" />` will capture the new template. 

What is this item group for?

Reply:
I didn't add it, when I added the file it created that line of code 😆 but it does seem like we don't need it because of the EmbeddedResource 👍